sonic_api.yaml

```diff
@@ -1255,6 +1255,124 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 #----------------------------------------------
+# VRF config API
+#----------------------------------------------
+  '/config/vrf/{vrf_id}':
+    post:
+      summary: Create a new VRF representing a virtual routing instance. If VRF exists, the attributes will be updated.
+      parameters:
+        - name: vrf_id
+          in: path
+          required: true
+          type: string
+          description: vrf_id can be a guid
```

Comment:
cannot be a guid, only integer, right?

Reply:
It can be anything depending upon the implementation. We currently use a guid to differentiate VRFs, thats why made it a string

Reply:
It will be good to define responsibility of the SONIC REST Implementation vs Caller. As the underlying platform does not support >4 bytes of VRF_ID, it will need to generate a unique id if GUID string is passed as input.